scripts/check-public-api-cli.ts

```diff
@@ -0,0 +1,14 @@
+import { createLogger } from '@sap-cloud-sdk/util';
+import { checkApiOfPackage, typeDescriptorPaths } from './check-public-api.js';
+
+const logger = createLogger('check-public-api');
+
+/*
+For a detailed explanation what is happening here have a look at `0007-public-api-check.md` in the implementation documentation.
+ */
+checkApiOfPackage(process.cwd()).catch(err => {
+  logger.error(err);
+  process.exit(1);
+});
+// const cwd = "C:\\Users\\I745181\\Documents\\AISDK\\ai-sdk-js\\packages\\gen-ai-hub\\dist";
```

Comment:
I suppose this was for testing purposes?

Reply:
Whoopsie
Yeah it was